src/Internal/Plutip/Server.purs

```diff
@@ -395,6 +401,12 @@ startPostgresServer pgConfig _ = do
     ]
     defaultSpawnOptions
   liftEffect $ killOnExit pgChildProcess
+  liftEffect $ onExit pgChildProcess \_ ->
```

Comment:
not sure what goes wrong here. If I try to remove `databaseDir` directly (commented out line 409) I get an `ENOTEMPTY` error and if I try to delete files first like here, I get an `ENOENT` error, meaning there are no files. Is it because I need to run the operation in `Aff`? tried to do this but haven't succeeded yet..